chrome/content/zotero/bindings/itembox.xml

```diff
@@ -2476,6 +2490,8 @@
 							return !hideTransforms;">
 						<menuitem label="&zotero.item.creatorTransform.nameSwap;"
 							oncommand="document.getBindingParent(this).swapNames();"/>
+						<menuitem label="&zotero.item.textTransform;"
+							oncommand="document.getBindingParent(this).textTransform(document.popupNode, 'title')"/>
```

Comment:
"Text Transform" isn't clear here. Let's go with a new string. (And can be title-cased. We can fix "Swap first/last names", which is sentence case for some reason.)